spec/repositories/submission_repository_spec.rb

```diff
@@ -109,4 +109,36 @@
       end
     end
   end
+
+  describe '#to_invite' do
```

Comment:
Assuming that you decide to calculate status dynamically and you have `confirmation_token_generated_at` and `confirmed_at` instead of `status` attribute:

these 3 objects would be enough for me:
* `to_invite` with high `rate_val` and `confirmation_token_generated_at: nil`
* `invited` with high `rate_val` and `confirmation_token_generated_at: 6.days.ago`
* `not_accepted` with too low `rate_val`

`SubmissionRepository#to_invite` should return only `to_invite` one.